### ISSUE TITLE
Does the color '#c411a6' break randomColor?

### ISSUE THREAD

**Issue by kuworking** [davidmerfield/randomColor#141]
I'm seeing that this code breaks the generation or `randomColor`, but I find this esoterical so it's likely something else

```js
  console.log(randomColor({
    count: 5,
    luminosity: 'light',
    hue: '#c411a6',
  }))
```

I do see it in a codesandbox

```js
import "./styles.css";
import randomColor from 'randomcolor'

console.log(randomColor({
  count: 5,
  luminosity: 'light',
  hue: '#ffffff',
}))

console.log(randomColor({
  count: 5,
  luminosity: 'light',
  hue: '#c411a6',
}))

document.getElementById("app").innerHTML = `
<h1>Hello Vanilla!</h1>
<div>
  We use the same configuration as Parcel to bundle this sandbox, you can find more
  info about Parcel 
  <a href="https://parceljs.org" target="_blank" rel="noopener noreferrer">here</a>.
</div>
`;
```

The hue value is from a previous randomly `randomColor`-generated array

![image](https://user-images.githubusercontent.com/4474004/102924586-2dd85b80-4492-11eb-9bf8-6e1c26c32a4a.png)

**Comment by daniegarcia254** [comment]
I'm having same issue using `#f9c2e3` as `hue` property.
It is as well a color previously generated with `randomColor` (using last library version)

**Comment by hmendezm** [comment]
Hi guys, is this fixed in the version 0.6.2? I have the same issue with the color #840052
best
Hmendez

**Comment by hmendezm** [comment]
I checked the version 0.6.2 that I have in my app and I noticed that itsmygit solution for hextohsb is not in this version.
